Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM amd64/ubuntu:latest
+
+# Get env variable from host.
+ARG _APPLET_PRIVATE_KEY
+
+# Install dependencies.
+RUN apt-get update && apt-get install -y make
+RUN apt-get install -y signify-openbsd
+RUN apt-get install -y wget
+RUN wget https://github.com/usbarmory/tamago-go/releases/download/tamago-go1.20.4/tamago-go1.20.4.linux-amd64.tar.gz
+RUN tar -xvf tamago-go1.20.4.linux-amd64.tar.gz -C /
+
+WORKDIR /build
+
+COPY . .
+
+# Set Tamago path for Make rule.
+ENV TAMAGO=/usr/local/tamago-go/bin/go
+
+# Set signing key path for Make rule.
+RUN echo 'untrusted comment: signify secret key' > armored-witness-applet.sec
+RUN echo $_APPLET_PRIVATE_KEY >> armored-witness-applet.sec
```

Comment:
Certainly not for this PR, but it might be worth a comment here to the effect of "this is temporary, the key is moving to a more secure environment" as writing it out to a file is very scary thing to do :)